subsys/caf/events/sensor_data_aggregator_event.c

```diff
@@ -10,9 +10,7 @@
 
 static void log_sensor_data_aggregator_event(const struct app_event_header *aeh)
 {
-	const struct sensor_data_aggregator_event *event = cast_sensor_data_aggregator_event(aeh);
-
-	APP_EVENT_MANAGER_LOG(aeh, "Send sensor buffer desc: %s", event->sensor_descr);
+	APP_EVENT_MANAGER_LOG(aeh, "Send sensor buffer");
```

Comment:
This change should not be needed.... if uart is a problem maybe we should just increase sampling period, etc

Reply:
or let's just disable this log in the sample

this code is common code so we should not change it for sample sake

Reply:
The problem here is that sensor_descr is kept in  net core flash and cannot be access by app core